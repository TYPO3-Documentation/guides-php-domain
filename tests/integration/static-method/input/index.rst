=================
PHP static method
=================

..  php:staticmethod:: doSomething(string $whatever): string

    Do something
