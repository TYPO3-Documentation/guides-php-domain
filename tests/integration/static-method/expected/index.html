<!-- content start -->
    <div class="section" id="php-static-method">
            <h1>PHP static method</h1>

            <dl class="php method">
    <dt class="sig sig-object php" id="dosomething"><span class="sig-name modifier"><span class="pre">static</span></span>

<span class="sig-name descname"><span class="pre">doSomething</span></span>
<span class="sig-paren">(</span>
<em class="sig-param"><span class="pre">string $whatever</span></em><span class="sig-paren">)</span>
<em class="sig-returns"><span class="pre">: string</span></em>
<a class="headerlink" href="#dosomething" title="Permalink to this definition">Â¶</a>
    </dt>
    <dd>
        <p>Do something</p>
        </dd>
</dl>

    </div>

<!-- content end -->
